draft-ietf-taps-interface.md

```diff
@@ -126,13 +126,13 @@ informative:
 
 --- abstract
 
-This document describes an abstract programming interface to the transport
+This document describes an abstract programming interface, API, to the transport
```

Comment:
This now looks as if API was an abbreviation for "abstract programming interface", which it really isn't  :)   so I suggest to write instead: "This document describes an abstract application programming interface, API, to the transport..."

Reply:
Yes, I would think that would be good.